text/0000-modular-crypto-service.md

```diff
@@ -79,29 +79,170 @@ explain more fully how the detailed proposal makes those examples work.
 [**WIP**] along with previous feature examples
 ## Detail proposal for BCCSP
 ```
-- sw(adjust)
-- sm2(sample for new crypto standard)
-- util(adjust/new added as class loader)
+- sw(adjust/new added as class loader)
```

Comment:
Avoid to use java language concept in golang project.

Reply:
em... ok will update it with design pattern logic.